### PR TITLE
Get rid of builders.hpp and subgraph_builders.hpp includes

### PR DESCRIPTION
**Pull request by olpipi**
### Details:
 - Get rid of builders.hpp and subgraph_builders.hpp includes

### Tickets:
 - [CVS-117139](https://jira.devtools.intel.com/browse/CVS-117139)

### CODE REVIEW

**Comment by olpipi** [comment]
Is blocked by openvino_contrib update https://github.com/openvinotoolkit/openvino_contrib/pull/811

**Review by vurusovs — APPROVED** [review]

**Review by p-durandin — APPROVED** [review]

**Review by jane-intel — APPROVED** [review]

**Comment by olpipi** [comment]
npu plugin build failed due to lack of deleted dependencies. So fix to npu repo have to be merged prior to this PR

**Comment by p-durandin** [comment]
Blocked by https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/8232

**Comment by p-durandin** [comment]
Next NPU PR https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/8563

File: src/plugins/intel_gpu/tests/functional/behavior/infer_request.cpp [hunk 1]
```diff
@@ -8,7 +8,6 @@
 #include "common_test_utils/node_builders/activation.hpp"
```
Comment (iefode) [inline src/plugins/intel_gpu/tests/functional/behavior/infer_request.cpp:8]: You have updated  `src/plugins/intel_cpu/thirdparty/mlas`. Please revert

Comment (olpipi) [reply src/plugins/intel_gpu/tests/functional/behavior/infer_request.cpp:8]: thx. I missed it

**Pull request merged**